### ISSUE TITLE
Make metadata shown in resource popovers and other resource previews curatable

### ISSUE THREAD

**Issue by j0hj0h** [hbz/oerworldmap#1518]
The information going into the preview for resources that is used in popovers on the map and probably in search results soon as well, should be prioritizable on the schema level.

[draft for resource preview](https://user-images.githubusercontent.com/15141949/36479831-6a54d6fa-170a-11e8-97dd-822c302673a1.png)

**Comment by karindr** [comment]
Good idea to show the tags so prominent so people will be encouraged to use tags for their own profiles.
